src/systemctl/systemctl.c

```diff
@@ -3375,10 +3375,8 @@ static int logind_check_inhibitors(enum action a) {
                 if (!sv)
                         return log_oom();
 
-                if (!pid_is_valid((pid_t) pid)) {
-                        log_error("Invalid PID %" PRIu32 ".", pid);
-                        return -ERANGE;
-                }
+                if ((pid_t) pid < 0)
+                        return log_error_errno(ERANGE, "Bad PID %"PRIu32": %m", pid);
```

Comment:
Hm, this would print "Bad PID 32767" or so. The format should be changed (not in this commit though).

Reply:
It took three of us, but eventually we decided to do the right thing.  Agreed, PID_FMT will make a better message.

Reply:
Amended, thanks for the review.